MAUI/Shimmer/built-in-views.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the Built-in types support of Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.
```

Comment:
description: Learn here about the Built-in types of support of Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.